src/components/Map/index.js

```diff
@@ -32,20 +44,27 @@ const Map = ({ neighborhoods, onSelect }) => (
             onSelect={onSelect}
           />,
         )}
+        {activeDatum && (
+          <LayerGroup key={activeDatum.id}>
+            <GeoJSON data={({ features: [activeDatum], type: 'FeatureCollection' })} {...activePathOptions} />
+          </LayerGroup>
+        )}
```

Comment:
I think just this will work: 

```
        {activeDatum && (
          <GeoJSON
            key={activeDatum.id}
            data={activeDatum}
            {...activePathOptions}
          />
        )}
```